shopper_ds/shopper.py

```diff
@@ -62,6 +62,13 @@ def safe_init_device(self):
     @state_attribute(
         bind=['ShopperInterlock'])
     def InterlockAlarm(self, alarm):
+        """
+        This attribute changes state of device, accordingly to ShopperInterlock
```

Comment:
Technically, this isn't a Tango attribute, only the decorator contains `attribute` in its name 😄